src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/StringMarshaller.Ansi.cs

```diff
@@ -1,149 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System;
-using System.Collections.Generic;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
-
-using static Microsoft.Interop.MarshallerHelpers;
-
-namespace Microsoft.Interop
-{
-    public sealed class AnsiStringMarshaller : ConditionalStackallocMarshallingGenerator
```

Comment:
We've removed all usages of `ConditionalStackallocMarshallingGenerator` now, do we want to delete it as well?